docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java

```diff
@@ -201,8 +202,21 @@ public CreateContainerCmd createContainerConfig(CreateContainerCmd containerConf
         if (dnsHosts.length > 0)
             containerConfig.withDns(dnsHosts);
 
-        if (volumesFrom != null && !volumesFrom.isEmpty())
-            containerConfig.withVolumesFrom(VolumesFrom.parse(volumesFrom));
+        if(volumesString!=null) {
+            for (String volume : splitAndFilterEmpty(volumesString)) {
+                try {
+                    containerConfig.withBinds(Bind.parse(volume));
+                } catch (IllegalArgumentException e) {
+                    containerConfig.withVolumes(Volume.parse(volume));
```

Comment:
Bind 'mounts' a volume from the host, 'Volume' creates a volume
Both adds parsers but didn't find common parser to detect wether we were talking of a bind `hostPath:ContainerPath[:Mode]` or a volume `ContainerPath`